reference/5.1/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -0,0 +1,190 @@
+> value 12, and `1.2345e-5L` has value 0, none of which is immediately obvious.
+
+## Numeric multipliers
+
+For convenience, integer and real literals can contain a numeric multiplier,
+which indicates one of a set of commonly used powers of 10. The numeric
+multiplier can be written in any combination of upper or lowercase letters.
+
+The multiplier suffixes can be used in combination with the `u`, `ul`, and `l`
+type suffixes.
+
+### Multiplier examples
+
+```
+PS> 1kb
+1024
+
+PS> 1.30Dmb
+1363148.80
+
+PS> 0x10Gb
+17179869184
+
+PS> 1.4e23tb
+1.5393162788864E+35
+
+PS> 0x12Lpb
+20266198323167232
+```
+
+## Numeric type accelerators
+
+To pair with the new suffixes that closely reflect C\# or F\# literal suffixes,
+I've also added a couple additional type accelerators that alias existing ones:
+
+PowerShell supports the following type accelerators:
+
+| Accelerator |         Note         |           Description            |
+| ----------- | -------------------- | -------------------------------- |
+| `[byte]`    |                      | Byte                             |
+| `[sbyte]`   |                      | Byte (unsigned)                  |
+| `[Int16]`   |                      | 16-bit integer                   |
+| `[UInt16]`  |                      | 16-bit integer (unsigned)        |
+| `[Int32]`   |                      | 32-bit integer                   |
+| `[UInt32]`  |                      | 32-bit integer (unsigned)        |
+| `[int]`     | alias for `[int32]`  | 32-bit integer (unsigned)        |
+| `[Int64]`   |                      | 64-bit integer                   |
+| `[UInt64]`  |                      | 64-bit integer (unsigned)        |
+| `[long]`    | alias for `[int64]`  | 64-bit integer (unsigned)        |
+| `[bigint]`  |                      | See [BigInteger Struct][bigint]  |
+| `[single]`  |                      | Single precision floating point  |
+| `[float]`   | alias for `[single]` | Single precision floating point  |
+| `[double]`  |                      | Double precision floating point  |
+| `[decimal]` |                      | 128-bit floating point           |
+
+### Working with other numeric types
+
+To work with any other numeric types you must use type accelerators, which is
+not without some problems. For example, high integer values are always parsed
+as double before being cast to any other type.
+
+```
+PS> [bigint]111111111111111111111111111111111111111111111111111111
+111111111111111100905595216014112456735339620444667904
+```
+
+The value is parsed as a double first, losing precision in the higher ranges.
+To avoid this problem, enter values as strings and then convert them:
+
+```
+PS> [bigint]'111111111111111111111111111111111111111111111111111111'
+111111111111111111111111111111111111111111111111111111
+```
+
+## Examples
+
+The following table contains several examples of numeric literals and lists
+their type and value.
```

Comment:
Use of "the following" should end the statement with a colon.

Reply:
Fixed